pyannote/audio/core/task.py

```diff
@@ -255,24 +316,348 @@ def __init__(
         self._metric = metric
 
     def prepare_data(self):
-        """Use this to download and prepare data
-
-        This is where we might end up downloading datasets
-        and transform them so that they are ready to be used
-        with pyannote.database. but for now, the API assume
-        that we directly provide a pyannote.database.Protocol.
+        """Use this to prepare data from task protocol
 
         Notes
         -----
-        Called only once.
+        Called only once on the main process (and only on it), for global_rank 0.
+
+        After this method is called, the task should have a `prepared_data` attribute
+        with the following dictionary structure:
+
+        prepared_data = {
+            'protocol': name of the protocol
+            'audio-path': array of N paths to audio
+            'audio-metadata': array of N audio infos such as audio subset, scope and database
```

Comment:
"audio-metadata" is listed twice, with 2 different descriptions.
Which one is correct?

Reply:
The first is more complete and contains more information about the contents of this key.